net/trojan/Portfile

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        trojan-gfw trojan 1.15.1 v
+revision            0
+categories          net
+maintainers         {@i0ntempest me.com:szf1234} openmaintainer
+platforms           darwin
+license             GPL-3
+
+description         An unidentifiable mechanism that helps you bypass network restrictions.
+long_description    Trojan features multiple protocols over TLS to avoid both \
+                    active/passive detections and ISP QoS limitations. It doesn't rely on \
+                    strong encryption or obfuscation\; instead it imitates the most \
+                    common protocol, HTTPS, so firewalls won't see anything other than \
+                    normal HTTPS traffic.
+
+checksums           rmd160  b36ac226c6d1cd56c37cd1ef675e0c11fb2d5f19 \
+                    sha256  8627a7cc6e1ebaca93a93cb19e4cc8aac853e1602f117c0470e43db916fded68 \
+                    size    48139
+
+depends_lib-append  port:boost \
+                    path:lib/libssl.dylib:openssl
+
+configure.args-append \
+                    -DCMAKE_INSTALL_SYSCONFDIR=${prefix}/etc -DDEFAULT_CONFIG=${prefix}/etc/trojan/config.json -DENABLE_MYSQL=OFF
+
+pre-configure {
+    reinplace "s|/etc/trojan/config.json|${prefix}/etc/trojan/config.json|g" ${worksrcpath}/docs/trojan.1
+}
+
+variant mysql57 conflicts mysql8 mariadb55 mariadb102 description {Enable authencation with a database using MySQL 5.7} {
```

Comment:
This long is also too long. Moving the description to the next line should work.